app/assets/javascripts/hyrax/nav_safety.js

```diff
@@ -4,7 +4,9 @@
 // - Add the nav-safety class to the form element.
 
 Blacklight.onLoad(function() {
+  var clickedTab;
   $('.nav-safety-confirm').on('click', function(evt) {
+    clickedTab = $(this).attr('href');
```

Comment:
Just a thought @Dananji in the future, for shared convention if referring to a jQuery variable, how about doing `var $clickedTab = ...` instead of `var clickedTab = ...`?

Reply:
Yes :+1: Thanks!